src/main/java/io/reactivesw/inventory/infrastructure/configuration/EventConfig.java

```diff
@@ -6,15 +6,33 @@
 import org.springframework.context.annotation.Configuration;
 
 /**
- * event config.
+ * Event config.
  */
 @Configuration
 @Data
 public class EventConfig {
 
   /**
-   * google cloud project id.
+   * Google cloud project id.
    */
   @Value("${io.reactivesw.message.google.project.id}")
   private String googleCloudProjectId;
+
+  /**
+   * Inventory reserved topic name.
+   */
+  @Value("${io.reactivesw.message.topic.inventoryreserved.name}")
+  private String inventoryReservedName;
+
+  /**
+   * Inventory reserved topic version.
+   */
+  @Value("${io.reactivesw.message.topic.inventoryreserved.version}")
+  private Integer inventoryReservedVersion;
+
```

Comment:
if this service has other event message, you have to change this name or add a new version? 
eventVersion will be better?